tool-plugins/vscode/src/docs/renderer.ts

```diff
@@ -0,0 +1,46 @@
+import { Uri, ExtensionContext } from 'vscode';
+import * as path from 'path';
+
+export function render(context: ExtensionContext): string {
+    let composerJsSrc;
+    if (process.env.COMPOSER_DEBUG === "true") {
```

Comment:
The condition should change to debug false. and try to make this a common component where the script section can be contributed.

Reply:
Changed the condition. Seems the existing util function has the ability to have additional scripts contributed. But I've used an onload function as well. Shall we do the refactoring to use a common component later. We have to consider how chunking might affect this as well.

Reply:
+1